spark/src/test/scala/org/apache/comet/CometArrayExpressionSuite.scala

```diff
@@ -259,6 +259,25 @@ class CometArrayExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelp
     }
   }
 
+  test("array_union") {
```

Comment:
Thanks for the unit test @drexler-sky . There is also a "array_union functions" test in Spark's DataFrameFunctionsSuite so we've got pretty good coverage for this!